app/addons/documents/assets/less/sidenav.less

```diff
@@ -16,176 +16,142 @@
-      .accordion-list-item:hover + div.add-dropdown .dropdown-toggle{
-        color: white;
-      }
-      .accordion-list-item p{
-        .transition(all 0.25s linear);
-      }
-      .design-doc-name {
-        cursor: pointer;
-        margin: 0;
-        color: @linkColor;
-        span {
-          width: @sidebarWidth - 30px;
-          display: block;
-          padding: 10px 13px 10px 36px;
-          text-overflow: ellipsis;
-          overflow: hidden;
-        }
-      }
-      .fonticon-play{
-        font-size: 12px;
-        top: 12px;
-        left: 12px;
-      }
-      &.down .fonticon-play {
-        .rotate(90deg);
-      }
-      .add-dropdown{
-        right: 15px;
-      }
-      .dropdown-toggle:hover{
-        color: @linkColor;
-      }
     }
-    > ul.accordion-body{
-      > li {
-        cursor: pointer;
-        a{
-          border-top: none;
-          &.accordion-header{
-            padding: 8px 5px 8px 36px;
-          }
-        }
-        .toggle-view{
-          padding: 4px 5px 4px 36px;
-        }
-        .fonticon-play{
-          font-size: 8px;
-          top: 11px;
-          left: 15px;
-        }
-        .fonticon:before{
-          margin-right: 6px;
-          font-size: 20px;
-          top: 3px;
-          position: relative;
-        }
-      }
 
-      li {
-        > a.down .fonticon-play {
-          .rotate(90deg);
-        }
-      }
+    .accordion-body {
+      color: #eee;
+      margin-left: 0;
 
       li {
+        cursor: pointer;
+        &.active > a {
+          background-color: rgba(255, 0, 0, 0.1);
+        }
         > a.down .fonticon-play {
           .rotate(90deg);
         }
-
-        &:hover {
-          .fonticon-play {
-            color: white;
-
+        &:hover .fonticon-play {
+          color: #dddddd;
+        }
+        a {
+          font-size: 14px;
+          background-color: rgba(0, 0, 0, 0.02);
+          &:hover {
+            color: #fff;
+            text-decoration: none;
+            background-color: @darkRed;
+          }
+          border-top: none;
+          &.accordion-header{
+            padding: 8px 5px 8px 36px;
           }
         }
-
       }
+    }
+
+    .toggle-view {
+      padding: 4px 5px 4px 36px;
+    }
+    .fonticon-play {
+      font-size: 8px;
+      top: 11px;
+      left: 15px;
+      color: @subListGray;
+      position: absolute;
+      display: block;
+      vertical-align: bottom;
+      .transition(all 0.25s linear);
+    }
+    .fonticon:before {
+      margin-right: 6px;
+      font-size: 20px;
+      top: 3px;
+      position: relative;
+    }
+    .accordion-header:hover .fonticon-play {
+      color: white;
+    }
+  }
 
+  .design-doc-name {
+    cursor: pointer;
+    margin: 0;
+    color: #222222;
+    span {
+      width: @sidebarWidth - 30px;
+      display: block;
+      padding: 10px 13px 10px 36px;
+      text-overflow: ellipsis;
+      overflow: hidden;
     }
   }
 }
+
+.index-group-header {
```

Comment:
Fair point! But I'd argue it's okay, actually. It's a pretty descriptive (specific) class name and it's the namespacing that ends up getting us in trouble with deep selectors + difficulty overriding them later. I did a search over the code base to ensure uniqueness (I always do when adding new selectors).